src/main/java/com/google/sps/servlets/StoreRatingServlet.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/store-rating")
+public class StoreRatingServlet extends HttpServlet {
```

Comment:
Tests for StoreRatingServlet and DrinkRatingServlet?

Reply:
I haven't made them yet. I thought making tests would occur at a later date

Reply:
In general, it's easiest and most efficient to write tests along with the code (when problem context is at its highest), and so submit each pull request along with tests. If there's a reason for deciding to add tests in a separate pull request, good to document this so peers understand the decision.

Reply:
Ok.